.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,29 @@
 <!--
-Thank you for contributing to manim!
-
-Please ensure that your pull request works with the latest
-version of manim from this repository.
+Thanks for your contribution to ManimCommunity!
+
+Before filling in the details, ensure:
+- Your local changes are up-to-date with ManimCommunity/manim
+
+- The title of your PR starts with at least one capitalized acronym:
+  API: an API breaking change
+  CI: change related to building manim
+  BUG: bug fix
+  DEP: deprecation, or removal of a deprecated object
+  DEV: development tool or utility
+  DOC: documentation
+  ENH: enhancement or new feature
+  MNT: maintenance (refactoring, typos, etc.)
+  REV: revert an earlier commit
+  STY: style fix (whitespace, PEP8)
+  TST: addition or modification of tests
+  REL: manim release
```

Comment:
Personally, I don't like these acronyms too much, as it is another learning step as a contributor, and it would take some time to enforce the usage of them.
Additionally,  I think they are a bit redundant with the pr labels.
But in case we don't have a good alternative, I am fine with using them

Reply:
I was just modeling after NumPy... after some quick research into *why* they do this in their repo, it's because they have an [automatic labeler workflow](https://github.com/numpy/numpy/blob/main/.github/workflows/labeler.yml). I agree this can be removed. The [changelog generator script](https://github.com/numpy/numpy/blob/main/tools/changelog.py) NumPy uses doesn't actually take advantage of these acronyms either which seems like a lost opportunity.